makefile.gen

```diff
@@ -10,18 +10,20 @@ include $(GDK)/common.mk
 SRC := src
 RES := res
 INCLUDE := inc
-
+OUT := out
 
 SRC_C= $(wildcard *.c)
 SRC_C+= $(wildcard $(SRC)/*.c)
 SRC_C+= $(wildcard $(SRC)/*/*.c)
 SRC_C+= $(wildcard $(SRC)/*/*/*.c)
 SRC_C:= $(filter-out $(SRC)/boot/rom_head.c,$(SRC_C))
+ifeq ($(SRC_C),)
+SRC_C=$(SRC)/main.c
+endif
 SRC_S= $(wildcard *.s)
 SRC_S+= $(wildcard $(SRC)/*.s)
 SRC_S+= $(wildcard $(SRC)/*/*.s)
 SRC_S+= $(wildcard $(SRC)/*/*/*.s)
-SRC_S:= $(filter-out $(SRC)/boot/sega.s,$(SRC_S))

```

Comment:
Why it was removed ? this line is important to prevent embedding linking sega.s into the lib

Reply:
I'm reverting this.  See also my other replies.

Reply:
Done.